packages.yml

```
@@ -1,6 +1,8 @@
 packages:
-- package: fivetran/fivetran_utils
-  version: [">=0.4.0", "<0.5.0"]
+# - package: fivetran/fivetran_utils
+#   version: [">=0.4.0", "<0.5.0"]
+- git: https://github.com/fivetran/dbt_fivetran_utils.git
+  revision: feature/add-lookback-macro
```

Update before release
```suggestion
- package: fivetran/fivetran_utils
  version: [">=0.4.0", "<0.5.0"]
```

---

This was reverted.